core/src/main/java/greencity/controller/OrderController.java

```diff
@@ -175,7 +176,8 @@ public ResponseEntity<FondyOrderResponse> processOrder(
         @Valid @PathVariable("id") Optional<Long> id) {
         if (id.isPresent()) {
             OrderDetailStatusDto orderDetailStatusDto = ubsManagementService.getOrderDetailStatus(id.get());
-            if (orderDetailStatusDto.getPaymentStatus().equals("PAID")) {
+            if (orderDetailStatusDto.getPaymentStatus().equals("PAID")
```

Comment:
please use `"PAID".equals(orderDetailStatusDto.getPaymentStatus())` to ommit `NPE`
and please move `PAID` to the constant